sprocs/calculate_assessment_question_stats_for_assessment.psql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE FUNCTION
+    calculate_assessment_question_stats_for_assessment (
+        assessment_id_var bigint
+    ) RETURNS VOID
+AS $$
+BEGIN
+    PERFORM
+        calculate_assessment_question_stats(aq.id)
+    FROM
+        assessment_questions AS aq
+    WHERE
+        aq.assessment_id = assessment_id_var
+        AND aq.deleted_at IS NULL;
+
+    UPDATE assessments AS a
+    SET
```

Comment:
It would be ok to write `SET stats_last_updated = current_timestamp` all on one line here because it's a single statement. It's also ok as it is, however.